Lib/test/test_locale.py

```diff
@@ -533,6 +533,12 @@ def test_defaults_UTF8(self):
             if orig_getlocale is not None:
                 _locale._getdefaultlocale = orig_getlocale
 
+    def test_getencoding(self):
+        # Invoke getencoding to make sure it does not cause exceptions.
+        enc = locale.getencoding()
```

Comment:
I suggest to test the type: add ``self.assertIsInstance(enc, str)``.

Maybe also ensure that the string is not empty? add ``self.assertNotEqual(enc, "")``.